src/chrome/client/chromeDebugAdapter/cdaConfiguration.ts

```diff
@@ -53,9 +53,6 @@ export class ConnectedCDAConfiguration implements IConnectedCDAConfiguration {
             launchUrl = utils.pathToFileURL(this.args.file);
         } else if (this.args.url) {
             launchUrl = this.args.url;
-        } else {
```

Comment:
Would it make more sense to change this to else if (isUndefined(config.runtimeExecutable)) instead of removing this branch?

Reply:
`runtimeExecutable` is only known about by chrome-debug. How can it participate in this validation?

Reply:
You are right. 
This code should live in chrome-debug. Requested url doesn't make sense for node.
Can you add a // TODO: Move this to chrome-debug and re-enable this throw new error with else if (isUndefined(config.runtimeExecutable)) throw new Error(`You must specify either file or url to launch Chrome against a local file or a url. None were specified. ` + `The specified parameterse are: ${JSON.stringify(this.args)}`);